docs/core/diagnostics/runtime-contention-events.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Thread Contention Events"
+description: See ETW events that collect information specific to the monitor lock contentions.
+ms.date: "09/27/2020"
+ms.topic: reference
+helpviewer_keywords:
+  - "thread contention events" [.NET Core]"
+  - "ETW, EventPipe, LTTng contention events (CoreCLR)"
+ms.assetid: 167a4459-bb6e-476c-9046-7920880f2bb5
+---
+
+# Contention Events
+
+These runtime events capture information about monitor lock contentions such as with Monitor.Enter or the C# lock keyword.
```

Comment:
We should improve this introduction with more content. What is this content about? Why do I care to read it? What benefit does it give me? Also, you might want to consider adding a clause at the end that says "For more information about diagnostics, see <link to diagnostics overview>"

Reply:
Should Moniter.Enter be protected with a code block? Otherwise, it will be translated into other languages. If this is a non translatable ETW field/category/value/code, protect it.

```suggestion
These runtime events capture information about monitor lock contentions such as with `Monitor.Enter` or the C# lock keyword.
```